src/sphinx_book_theme/assets/styles/pages/_search.scss

```diff
@@ -3,45 +3,50 @@
  */
 
 // Turn off the extra search UI at the top of the page
-#search-documentation {
-  display: none;
-  & ~ p,
-  ~ form {
+.bd-search-container {
```

Comment:
Could we re-use the icon search pattern in the pydata theme instead of defining our own text field for it? For example, we could put the search button in the sidebar along with any icons that were site-wide and not specific to the page:

![image](https://user-images.githubusercontent.com/1839645/189868863-de6f71e8-36f8-4515-951c-998d15743501.png)

Reply:
@choldgraf actually we are using the html defined in pydata only for this. Just adding our own styles for positioning it. https://github.com/pydata/pydata-sphinx-theme/blob/main/src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/search-field.html. But I am okay with using the pydata search pattern. Should we do it in a separate PR, as its more like a new enhancement? Or you want it here?